CHANGELOG.md

```diff
@@ -40,8 +40,8 @@ v0.36.0 (2023-08-30)
 - `otelcol.exporter.prometheus`: Set `include_scope_info` to `false` by default. You can set
   it to `true` to preserve previous behavior. (@gouthamve)
 
-- `prometheus.remote_write`: Set `retry_on_http_429` to `true` by default in the `queue_config` block.
-  You can set it to `false` to preserve previous behavior. (@wildum)
+- Set `retry_on_http_429` to `true` by default in the `queue_config` block.
+  Both static mode's `remote_write` and flow mode's `prometheus.remote_write` are affected. (@wildum)
```

Comment:
This is now in a wrong place. It's something that happens when we cut the release and we should look out for. It should go to unreleased changes, since the release was cut.